library/Freckle/App/Datadog.hs

```diff
@@ -16,31 +16,40 @@ module Freckle.App.Datadog
   -- * Higher-level operations
   , increment
   , counter
-  , guage
+  , gauge
```

Comment:
Do we need to keep `guage` for backwards compat? Or is this a major version bump?

Reply:
Good question.

I'd probably just major it. *Except*, we did get this into Stackage, so giving it a major would saddle us with our own `extra-dep` for a while.

In this case keep `guage = gauge {-# DEPRECATED -#}` is not a huge burden to prevent that.

On the other-other hand, `freckle-app` is probably going to see some churn for the next little bit, so we may be riding with the `extra-dep` for a while anyway.

Up to you.

Reply:
Sadly `{-# DEPRECATED -#}` is also a major bump within PVP. So all we can do is leave a comment. Dumb, I know.

> Deprecation. Deprecated entities (via a DEPRECATED pragma) SHOULD be counted as removed for the purposes of upgrading the API, because packages that use -Werror will be broken by the deprecation. In other words the new A.B SHOULD be greater than the previous A.B.

Reply:
Wow, that is totally dumb. It does say SHOULD, not MUST though...

Reply:
Oh that is true...

Reply:
I suspect it's only a SHOULD because you're also not supposed to release packages to Hackage with `-Werror`. I can't find the docs on that, but I'm 99% sure something will yell at you (`cabal` or even the actual `POST`).

Reply:
Yes you cannot release with `-Werror`